resources/config/snemo/demonstrator/simulation/primary_events/1.4/README.rst

```diff
@@ -0,0 +1,178 @@
+====================================================================
+Primary event generation for simulation (snemo::demonstrator)
+====================================================================
+
+:Authors: F. Mauger, Y. Lemière
+:Date:    2017-12-13
+
+.. contents::
+   :depth: 3
+..
+
+Presentation
+============
+
+This directory contains  the files needed to  describe several primary
+event generators as input for SuperNEMO simulation.
+
+  * Setup label is : ``snemo::demonstrator::simulation::primary_events``
+  * Version is : ``1.3``
+
+Files
+=====
+
+  * ``README.rst`` : this file,
+  * ``manager.conf`` : the   main  configuration file of the
+    genbb/decay0 manager object (from Bayeux/genbb_help),
+  * ``generators/`` : Directory for primary event generators definitions.
+  * ``variants/`` : Directory for variant definitions.
+  * ``_tools/`` : Utilities for mainteners.
+
+Check the configuration
+=======================
+
+First make sure the Bayeux and Falaise software is installed and setup:
+
+.. code:: sh
+
+   $ which bxvariant_inspector
+   ...
+   $ which bxgenbb_inspector
+   ...
+..
+
+
+Browse and edit primary event generation variant parameters and options
+===============================================================================
+
+1. From the build directory, generate documentation about variants:
+
+.. code:: sh
+
+   $ bxvariant_inspector \
+          --datatools::resource-path="falaise@$(pwd)/BuildProducts/share/Falaise-3.1.1/resources" \
+          --variant-config "@falaise:config/snemo/demonstrator/simulation/primary_events/1.3/variants/repository.conf" \
```

Comment:
As above, maybe change explicit versions to placeholders so changes can be confined to single place (e.g the Version in the Presentation section above.